apps/web/components/Organization/Slice/FeaturedArticles/FeaturedArticlesSlice.tsx

```diff
@@ -63,30 +65,30 @@ export const FeaturedArticlesSlice: React.FC<
             {(slice.automaticallyFetchArticles
               ? sortedArticles
               : slice.articles
-            ).map(
-              ({
-                title,
-                slug,
-                processEntry = null,
-                processEntryButtonText = null,
-              }) => {
-                const url = linkResolver('Article' as LinkType, [slug])
-                return (
-                  <FocusableBox key={slug} borderRadius="large" href={url.href}>
-                    <TopicCard
-                      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                      // @ts-ignore make web strict
-                      tag={
-                        (!!processEntry || processEntryButtonText) &&
-                        n(processEntryButtonText || 'application', 'Umsókn')
-                      }
-                    >
-                      {title}
-                    </TopicCard>
-                  </FocusableBox>
-                )
-              },
-            )}
+            ).map((article) => {
+              const url = linkResolver('Article' as LinkType, [article.slug])
+              return (
+                <FocusableBox
+                  key={article.slug}
+                  borderRadius="large"
+                  href={url.href}
+                >
+                  <TopicCard
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore make web strict
+                    tag={
+                      hasProcessEntries(article as Article) &&
```

Comment:
It seems that this change is impacting articles that have a stepper on them

Dev 
![image](https://github.com/island-is/island.is/assets/43557895/6bf96165-ed68-4eb1-bab4-3ac72435adc6)

This deployed change
![image](https://github.com/island-is/island.is/assets/43557895/85227da2-54a6-4d4d-a47b-6ceff0bd8a8b)

This field in the CMS sets the text and what it should be
![image](https://github.com/island-is/island.is/assets/43557895/87941908-5008-43a1-a7be-04253770187a)

Is there a way we can do this and still keep the old functionality present?

Reply:
@RunarVestmann Moving forward, all Process Entries will only be used for actual applications. Previously, the old code was only checking for the main Process Entry field on an Article however the search (correctly) looks for both the field and additionally the embeded content. This now checks everywhere the same as search so that the approach is the same throughout the site.

There's been a large internal discussion on how best to approach this topic considering Applications do not technically exist within the content system. Process Entry was meant to represent this however has grown in a way that it's currently being used for things that are not actual applications. This content strategy will change moving forward.

With that in mind, I am not sure we should be allowing a manual override of a tag in this case. An article either is or is not an application and that is based solely on the presence of Process Entry (field or embeded in rich text). This will ensure that the UI treatment is always the same.

Is the problem only a matter of labeling? If so, I can simply check if a: process entry exists and, if so, display either the dropdown value or the default global value for the tag. I would like to avoid on relying on the dropdown field being populated or not to dictate whether the tag should show. Feel free to ping me. This topic is some what complicated given the data structure.

Reply:
I've reviewed this. These articles do indeed Process Entry so this is not the desired outcome as the tag should show based on the util. I'll investigate why it's not detecting the  process entry.

Reply:
@RunarVestmann I found the issue. It's because these articles do not contain process entries at all. They have a Stepper content type embeded in the rich text. This unfortunately adds yet another complication to the mental map of what constitutes an "application".

In the meantime, I have changed the code to conditionally render based on the button text just for this section but this should get properly addressed once we implement the new CTA Link content type as logging in to see if you need a visa (step by step) is not an actual application. The UI should also be different to reflect this. I'll maintain a separate PR later for this new content type that should include this change as well. See latest commit.